walkeeper/src/send_wal.rs

```diff
@@ -104,7 +104,7 @@ impl SendWalHandler {
             Some(lsn_bytes),
             None,
         ]))?
-        .write_message(&BeMessage::CommandComplete(b"IDENTIFY_SYSTEM"))?;
+        .write_message(&BeMessage::CommandComplete(b"SELECT 1"))?;
```

Comment:
What's this?

Reply:
When I wrote code for handling GC request I realized that I have not correctly generate command completion tag. I looked through all places where BeMessage::CommandComplete is used and fixed them (actually there was just one in walkeeper).